manifests/params.pp

```diff
@@ -345,13 +345,16 @@
     $default_ssl_cert    = '/etc/ssl/certs/ssl-cert-snakeoil.pem'
     $default_ssl_key     = '/etc/ssl/private/ssl-cert-snakeoil.key'
     $ssl_sessioncache    = "\${APACHE_RUN_DIR}/ssl_scache(512000)"
-    if ($facts['os']['name'] == 'Ubuntu') or ($facts['os']['name'] == 'Debian' and versioncmp($facts['os']['release']['major'], '11') < 0) {
-      $php_version = $facts['os']['release']['major'] ? {
-        '9'     => '7.0', # Debian Stretch
-        '10'    => '7.3', # Debian Buster
-        '20.04' => '7.4', # Ubuntu Foccal Fossal
-        default => '7.2', # Ubuntu Bionic, Cosmic and Disco
-      }
+    $php_version = $facts['os']['release']['major'] ? {
+      '9'     => '7.0', # Debian Stretch
+      '10'    => '7.3', # Debian Buster
+      '11'    => '7.4', # Debian Bullseye
+      '20.04' => '7.4', # Ubuntu Foccal Fossal
+      '22.04' => '8.1', # Ubuntu Jammy
+      default => '7.2', # Ubuntu Bionic, Cosmic and Disco
+    }
+    if (($facts['os']['name'] == 'Ubuntu' and versioncmp($facts['os']['release']['major'], '22.04') < 0) or
+    ($facts['os']['name'] == 'Debian' and versioncmp($facts['os']['release']['major'], '11') < 0)) {
```

Comment:
I did create [`os_version_gte`](https://forge.puppet.com/modules/puppetlabs/stdlib/reference#os_version_gte) for this in the past, but never got around to using it. If you want, you could rewrite the logic here. Be sure to use at least stdlib 8.1.0 then since that fixed the logic.

Reply:
While that function looks like a good idea, doing this would require that a major release be made as we would need to bump the current lower stdlib version bound (it's currently4.13.1), so I think it may be best to leave as is for now.

Reply:
I get you may consider it too large, but normally I don't consider increasing the minimum version a reason for a major version bump. Though 8.1.0 is recentish for stdlib (~1 year old).